update-version.sh

```diff
@@ -139,6 +139,18 @@ get_new "wolfcrypt-py"
 update "wolfcrypt-py"
 
 
+printf "Checking version of wolfEngine to use..."
+get_current "wolfengine"
```

Comment:
You should create a function for these three lines. Too  many replications begging for typos.

```
function check_and_update() {
   get_current "$1"
   get_new "$1"
   update "$1"
}
```